### ISSUE TITLE
tar is broken on windows, include bsdtar instead?

### ISSUE THREAD

**Issue by wooger** [bmatzelle/gow#222]
Many common uses of the tar command are broken on windows, including anything with -z option.

Could you include the bsdtar command (present in gnuwin32) which solves this issue and works properly for most use cases?

You could even symlink a tar command to bsdtar.exe to avoid confusion, as I've done.

**Comment by jeeger** [comment]
Yes, I just came across this error. tar with absolute paths is unsusable, because it takes drive letters as hostnames for remote connections. bsdtar fixes this.
